### ISSUE TITLE
app-check : android 

### ISSUE THREAD

**Issue by physedo** [capawesome-team/capacitor-firebase#297]
imports:

```
import { FirebaseAppCheck, GetTokenResult, InitializeOptions, GetTokenOptions} from '@capacitor-firebase/app-check';
import {  initializeApp, provideFirebaseApp, getApp } from '@angular/fire/app';
import {

  initializeAppCheck,
  CustomProvider,
  provideAppCheck,
  AppCheckToken, 
} from '@angular/fire/app-check';
```
``

app.module.ts:
`
```
provideAppCheck(  () => {
      const options: InitializeOptions = {
        debug: false,
        isTokenAutoRefreshEnabled: true
      };
     FirebaseAppCheck.initialize(options);
      const androidprovider = new CustomProvider({
      
        getToken (): Promise<any> {
         
          return   FirebaseAppCheck.getToken();
        },
      });


      const app = getApp();
    return initializeAppCheck(app, {provider:androidprovider})
  }
    ),
````

debug is false and when it is true also
You can still debug it
It works with debug on android
But when I publish it on google play it does not work

**Comment by robingenz** [comment]
Thank you for your request. 
Please fill out the issue template correctly so that I have all information I need.

**Comment by robingenz** [comment]
Closed as duplicate of #298
